src/app/courses/course-item/course-item.component.html

```diff
@@ -1,6 +1,6 @@
-<div class="course-item">
+<div class="course-item" [appCreationdatestatus]="course.dateCreation" [ngClass]="{'course-item--rated': course.topRated}">
```

Comment:
when you have lots of bindings, you can make each one on a new line:
```html
<div class="course-item"
    [appCreationdatestatus]="course.dateCreation"
    [ngClass]="{'course-item--rated': course.topRated}"> ...
```
it will improve readability.
Also you can separate significant html blocks by adding an emty line between them.